.github/workflows/reusable-performance.yml

```diff
@@ -315,20 +315,13 @@ jobs:
       - name: Set commit details
         # Only needed when publishing results.
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/trunk' && ! inputs.memcached }}
-        uses: actions/github-script@60a0d83039c74a4aee543508d2ffcb1c3799cdea # v7.0.1
-        id: commit-timestamp
-        with:
-          github-token: ${{ secrets.GITHUB_TOKEN }}
-          script: |
-            const commit_details = await github.rest.git.getCommit({ owner: context.repo.owner, repo: context.repo.repo, commit_sha: context.sha });
-            return parseInt((new Date( commit_details.data.author.date ).getTime() / 1000).toFixed(0))
+        run: echo "COMMITTED_AT=$(git show -s $GITHUB_SHA --format='%cI')" >> $GITHUB_ENV
```

Comment:
`git show -s $GITHUB_SHA --format='%cI'` returns an ISO date string like `2024-12-19T14:30:02+01:00`, but the previous code here and the `log-results.js` script clearly expect to pass an integer (timestamp value, number of seconds)

Reply:
Gutenberg project passed ISO date string.

https://github.com/WordPress/gutenberg/blob/trunk/.github/workflows/performance.yml#L103
https://github.com/WordPress/gutenberg/blob/trunk/bin/log-performance-results.js#L44

GB Dashboard show correct date formate for commit not Core so we should pass date formate as per GB project.

GB project API responce: https://www.codevitals.run/api/evolution/2?limit=10

WP project API responce: https://www.codevitals.run/api/evolution/14?limit=10

Reply:
Then `timestamp` is a really bad name for a date string 🙃

Reply:
In https://github.com/WordPress/wordpress-develop/pull/8072/commits/2b4c7688324b8adf053c2080c7b74cc9e69ceecd name updated to `date`